app/javascript/dashboard/components/layout/Sidebar.vue

```diff
@@ -144,18 +144,24 @@ export default {
         newLink: true,
         key: 'inbox',
         cssClass: 'menu-title align-justify',
-        toState: frontendURL('settings/inboxes'),
+        toState: frontendURL(
+          `account/${Auth.getCurrentUser().account_id}/settings/inboxes`
```

Comment:
Not a big fan of `snake_cases` on FE side, Any way we could change this to `accountId `